p3/data/_validation.py

```diff
@@ -1,49 +1,48 @@
-# Copyright (c) 2022-2023 Intel Corporation
-# SPDX-License-Identifier: MIT
-
 import json
 import jsonschema
-import pkgutil
-
+import os
 
-def _validate_coverage_json(json_string: str) -> object:
+def _validate_coverage_data(coverage_data) -> object:
```

Comment:
I think it might be a good idea to be explicit that we're intending to accept any data type here:
```suggestion
def _validate_coverage_data(coverage_data: Any) -> object:
```